src/cityreader/cityreader.py

```diff
@@ -16,11 +21,16 @@
 # should not be loaded into a City object.
 cities = []
 
+import csv
 def cityreader(cities=[]):
+  with open('./src/cityreader/cities.csv') as csvfile:
+    reader = csv.reader(csvfile)
```

Comment:
You don't need to put the full path only the file name since the file resides in the same folder as this one.